vue/src/modules/Workspace/article/ArticleView.vue

```diff
@@ -0,0 +1,202 @@
+<template>
+  <div id="article-view" class="page">
+    <div class="page-header">
+      <back-button/>
+      <b-breadcrumb :items="breadCrumb" class="d-none d-sm-flex"></b-breadcrumb>
+      <b-dropdown variant="link" no-caret right v-if="isFolder">
+        <template v-slot:button-content>
+          <b-button size="sm" pill variant="link">
+            <i class="fas fa-plus-circle fa-3x"></i>
+          </b-button>
+        </template>
+        <b-dropdown-item v-if="lastFolder && multipleFolders"
+                         @click="showModal()">
+          <i class="fas fa-plus"></i>
+          {{ $t('Create new folder') }}
+        </b-dropdown-item>
+        <b-dropdown-item @click="showModal(true)">
+          <i class="fas fa-plus"></i>
+          {{ $t('Create new article') }}
+        </b-dropdown-item>
+      </b-dropdown>
+      <AddToFavourites :name="favouriteName"/>
+    </div>
+    <div class="page-content">
+      <content-spinner :show="loading" :text="$t('Loading...')" class="h-100"/>
+      <b-card no-body class="mb-1" v-if="isFolder">
+        <b-card-header header-tag="header" class="p-1" aria-controls="collapse"
+                       :aria-expanded="visible ? 'true' : 'false'" @click="visible = !visible">
+          <i v-if="!visible" class="fas fa-angle-double-down fa-2x"></i>
+          <i v-else class="fas fa-angle-double-up fa-2x"></i>
+        </b-card-header>
+        <b-collapse id="collapse" v-model="visible">
+          <b-card-body>
+            <div class="row">
+              <div class="col-md-4 col-sm-12">
+                <b-media class="article-header align-items-center">
+                  <b-img v-if="currentArticle.image_url" class="article-image" :src="currentArticle.image_url"/>
+                  <i v-else class="fas fa-folder-open fa-4x"/>
+                </b-media>
+              </div>
+              <div class="col-md-8 col-sm-12">
+                <div>
+                  <h3 class="mt-0">{{ currentArticle.title }}</h3>
+                </div>
+                <div v-html="currentArticle.body"></div>
+              </div>
+            </div>
+          </b-card-body>
+        </b-collapse>
+      </b-card>
+      <div class="p-3">
+        <div class="row" v-if="lastFolder && isFolder && multipleFolders">
+          <div class="col-sm-6 col-lg-4">
+            <h4 class="border-bottom pb-1 mb-3">{{ $t('Articles') }}</h4>
+            <div class="article-list">
+              <content-spinner :show="loading" :text="$t('Loading...')" class="h-100"/>
+              <no-data :model="filteredArticles" :loading="loading" :text="$t('There are no articles')"></no-data>
+              <div class="article-list">
+                <ArticleChildItem
+                  v-for="(article, index) in filteredArticles" :index="index" :model="article" :key="article.id"/>
```

Comment:
@skavt  Do not use directly folder.id as key. It may not be unique in the page, use itself and some unique string name combination.